docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -348,39 +334,27 @@ Open the **Control** tab of your mock robot to see the robot's arm positions cha
 
 Now that you have your mock robotic arm working, add a mock sub-part to your robot.
```

Comment:
This should mention the motor here as well because otherwise it's a bit like "ok but why?"

Reply:
done